### PR TITLE
Custom admin hook

### PR DESCRIPTION
**Pull request by Advayp**
Author: Advay

## What changes were made?

Custom hook to fetch all admins. 

## Why are these changes important/necessary?

Help complete the functionality of the report dashboard.

## Merge After
- #95

### CODE REVIEW

File: src/services/member.ts [hunk 1]
```diff
@@ -116,3 +116,16 @@ export async function isCurrentMemberVerified(): Promise<boolean> {
     )
     .catch(() => false);
 }
+
+export const getAllAdmins = async (): Promise<Member[]> => {
+  const url = `/members/admin/`;
+
+  const res = await api.get(url);
+
+  devPrint('res', res);
```
Comment (elimelt) [inline src/services/member.ts:125]: nit: let's try to either avoid adding logs or add a more descriptive name going forward, just so the console isn't crowded by a bunch of `res: <>` logs during development

**Review by elimelt — APPROVED** [review]
LGTM, let's try to keep console clean or at least more decipherable

**Comment by elimelt** [comment]
feel free to merge after the error is fixed. looks like it was just a merge error

File: src/services/member.ts (hunk 1 above)
Comment (Advayp) [reply src/services/member.ts:125]: Got it. I got rid of the ones I recently added.

**Pull request merged**